mock-core-configs/etc/mock/templates/anolis-7.tpl

```diff
@@ -0,0 +1,50 @@
+# This list is taken from 'epel-7-x86_64' @buildsys-build group, minus the
+# 'epel-*' specific stuff.
+config_opts['chroot_setup_cmd'] = 'install bash bzip2 coreutils cpio diffutils findutils gawk gcc gcc-c++ grep gzip info make patch redhat-rpm-config rpm-build sed shadow-utils tar unzip util-linux which xz'
+
+config_opts['dist'] = 'an7'  # only useful for --resultdir variable subst
+config_opts['releasever'] = '7'
+config_opts['package_manager'] = 'yum'
+config_opts['description'] = 'Anolis 7'
+
+config_opts['yum.conf'] = """
+[main]
+keepcache=1
+debuglevel=2
+reposdir=/dev/null
+logfile=/var/log/yum.log
+retries=20
+obsoletes=1
+gpgcheck=0
+assumeyes=1
+syslog_ident=mock
+syslog_device=
+metadata_expire=0
+mdpolicy=group:primary
+best=1
+protected_packages=
+user_agent={{ user_agent }}
+
+# repos
+
+[os]
+name=AnolisOS-7 - os
+baseurl=http://mirrors.openanolis.cn/anolis/7/os/$basearch/os
+enabled=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-ANOLIS
```

Comment:
Please add the key to https://github.com/xsuchy/distribution-gpg-keys/ and then use the path in the form of file:///usr/share/distribution-gpg-keys/...

Reply:
And please bump up mock-core-configs.spec to requires distribution-gpg-keys >= 1.74